### ISSUE TITLE
PHP Warnings on Exclude rules page

### ISSUE THREAD

**Issue by slaFFik** [xwp/stream#1031]
# Bug Report
WordPress Multisite 5.2.2.

When opening this page `/wp-admin/network/admin.php?page=wp_stream_network_settings` I see these errors:

```
Warning: Illegal string offset 'exclude_row' in /.../wp-content/plugins/stream/classes/class-settings.php on line 846

Warning: Illegal string offset 'exclude_row' in /.../wp-content/plugins/stream/classes/class-settings.php on line 848

Warning: Invalid argument supplied for foreach() in /.../wp-content/plugins/stream/classes/class-settings.php on line 848
```

## Expected Behavior

No errors.

## Actual Behavior

Describe what actually happens.

## Steps to Reproduce the Problem

1. Clean installation of the WPMS 5.2 on PHP 7.3
2. Network activate the plugin
3. Go to Exclude tab.

## Screenshots

![image](https://user-images.githubusercontent.com/282234/62248447-cb8c4000-b3f0-11e9-8d66-21cce68c8c8a.png)

## System Information

- Stream plugin version: 3.4.1
- WordPress version: 5.2.2
- PHP version: 7.3
- Browser: Chrome
- Computer operating system: Windows 10 Pro

**Comment by kasparsd** [comment]
This will be fixed along with #1010 in #1034.

**Comment by kasparsd** [comment]
Fixed in #1034 and will be included in the next point release.
